src/app/components/Link/index.stories.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import Link from './index';
+
+const link = name => ({
```

Comment:
This all looks good. We just need to note that because of the styling of this link, it is not visible against the Storybook layout, which has a white background. 
For now I believe we should look to merge this PR in as it is and then reassess this in a future PR.

Reply:
Agreed